helper/setup.py

```diff
@@ -52,7 +52,7 @@ def _copy_fonts(self):
 
 requirements = [
     'boto3', 'botocore', 'amazon-textract-response-parser>=0.1.40', 'amazon-textract-caller>=0.0.27',
-    'amazon-textract-overlayer>=0.0.10', 'amazon-textract-prettyprinter>=0.1.0', 'Pillow>=9.2.*', 'PyPDF2>=2.5.*'
```

Comment:
Why change PyPDF2

Reply:
since the version will be depreciated https://pypi.org/project/PyPDF2/

NOTE: The PyPDF2 project is going back to its roots. PyPDF2==3.0.X will be the last version of PyPDF2. Development will continue with [pypdf==3.1.0](https://pypi.org/project/pyPdf/).

Reply:
That is a good reason, @tb102122  :-)
Thx!